src/usdt/mod.rs

```diff
@@ -0,0 +1,187 @@
+use std::{collections::HashSet, ffi::{CStr, CString}, path::{Path, PathBuf}, ptr};
```

Comment:
Please don't use this syntax. I'd rather have separate use for `std::collections::..` `std::ffi::...` etc